master/internal/api_experiment.go

```diff
@@ -1302,7 +1300,7 @@ func (a *apiServer) PatchExperiment(
 				err = runCheckpointGCTask(
 					a.m.system, a.m.rm, a.m.db, taskID, modelExp.JobID, modelExp.StartTime,
 					taskSpec, modelExp.ID, modelExp.Config, checkpoints, []string{fullDeleteGlob}, true,
-					agentUserGroup, user, nil,
+					false, agentUserGroup, user, nil,
```

Comment:
I think this should be true?

We want to patch the checkpoint state here?

Reply:
yeah I can revert back -- doesn't affect tests either way. Which probably means that we should write a test to address this.